test/modules/io/writers/xviz-protobuf-writer.spec.js

```diff
@@ -0,0 +1,142 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+/* eslint-disable camelcase, no-unused-vars */
+import test from 'tape-catch';
+import {XVIZProtobufWriter, XVIZData, MemorySourceSink} from '@xviz/io';
+import {XVIZ_PROTOBUF_MESSAGE} from '@xviz/io';
+
+//
+//  Test data and cases for successful writer tests
+//
+const SAMPLE_METADATA = {
```

Comment:
Should this test include things we are having enum trouble with?

Reply:
The enum's issue is covered in parsing related tests in #503